yt/frontends/ramses/data_structures.py

```diff
@@ -744,6 +749,9 @@ def read_rhs(cast):
             for n in range(rheader['ncpu']):
                 dom, mi, ma = f.readline().split()
                 self.hilbert_indices[int(dom)] = (float(mi), float(ma))
+
+        if rheader['ordering type'] != 'hilbert' and self.bbox:
+            raise NotImplementedError('The ordering %s is not compatible with the `bbox` argument.' % rheader['ordering type'])
```

Comment:
Can you wrap this to 80 characters? You can have multiline strings inside of parens like this:

```
raise MyError(
    "This is a long error message that is longer than 80 characters "
    "so it is wrapped to appease Nathan because he doesn't like long "
    "lines in Python code")